### ISSUE TITLE
Loss value decrease first and then increase when I was training on my own dataset

### ISSUE THREAD

**Issue by HappyPeanuts** [ifzhang/FairMOT#341]
When I was training my data set, I found that my loss quickly dropped to a minimum of around 13 rounds, and then gradually increased. I tried to modify the learning rate to avoid going into a local optimal solution, but it still didn't work. In addition, in the process of loss hm_loss, wh_loss, off_loss and id_loss has been decreasing, what is the situation?

epoch: 9 |**loss 0.238135** | hm_loss 0.425285 | wh_loss 0.397233 | off_loss 0.164568 | id_loss 0.311685 | time 32.616667 | 
 epoch: 10 |**loss 0.120574** | hm_loss 0.418234 | wh_loss 0.390729 | off_loss 0.163374 | id_loss 0.251573 | time 33.666667 | 
 epoch: 11 |**loss 0.020429** | hm_loss 0.412478 | wh_loss 0.387238 | off_loss 0.162954 | id_loss 0.206802 | time 34.150000 | 
 epoch: 12 |**loss -0.060697** | hm_loss 0.408775 | wh_loss 0.385848 | off_loss 0.162448 | id_loss 0.176986 | time 34.100000 | epoch:    12  lr:0.000010
epoch: 13 |**loss -0.144822** | hm_loss 0.398481 | wh_loss 0.379990 | off_loss 0.160817 | id_loss 0.145303 | time 34.216667 | 
epoch: 14 |**loss -0.161400** | hm_loss 0.395515 | wh_loss 0.377716 | off_loss 0.160561 | id_loss 0.139927 | time 34.183333 |

**Comment by Maping1026** [comment]
hello, I meet the same question,  and the loss changes to a minus,  but I don't kown why, can you help me?

**Comment by AI-Hunter** [comment]
@HappyPeanuts 
sorry to comment in the close issue.
the same problem, any solution, please?
